### ISSUE TITLE
Component inheritance seems not to be fully supported with 

### ISSUE THREAD

**Issue by zoechi** [dart-lang/angular_analyzer_plugin#399]
I get this warning

```
warning: The bound output trigger does not exist on any directives or on the element (nonexist_output_bound at [myproject] lib/src/components/app/app_component.html:58)
```

![image](https://user-images.githubusercontent.com/405837/28667395-9e3238ac-72cb-11e7-807b-02dcb5837f82.png)

MaterialButtonComponent hierarchy
 
![image](https://user-images.githubusercontent.com/405837/28667402-a753be38-72cb-11e7-8c5e-75bd0cbe4999.png)

![image](https://user-images.githubusercontent.com/405837/28667410-b2ff6fac-72cb-11e7-91ec-96a7ee571185.png)

![image](https://user-images.githubusercontent.com/405837/28667422-bf12ad7c-72cb-11e7-8e19-7ec87d5af02c.png)

Directive is provided
![image](https://user-images.githubusercontent.com/405837/28667456-db118476-72cb-11e7-8897-85558ff68884.png)

**Comment by MichaelRFairhurst** [comment]
Is this using my fork with #397 ? Title seems incomplete.

**Comment by zoechi** [comment]
Just updated today according to the steps in the README.md

**Comment by MichaelRFairhurst** [comment]
Hey Gunter, sorry to leave this hanging so long.

Can you try clearing out ~/.dartServer/.analysis-driver/ and rerun?

**Comment by zoechi** [comment]
Should we close this? I don't even remember what project this was and I probably won't be able to do much with Angular for several months.

**Comment by MichaelRFairhurst** [comment]
Will do. It should definitely work, if you see any other issues please do file!
